cumulus_library/template_sql/code_system_pairs.sql.jinja

```diff
@@ -33,13 +33,11 @@ UNNEST({{ source.column_name }}.coding) AS t (row)
 SELECT *
 FROM (
     VALUES (
-        (
-            '{{ source.table_name }}',
-            '{{ source.column_name }}',
-            '',
-            '',
-            ''
-        )
+        '{{ source.table_name }}',
+        '{{ source.column_name }}',
+        '',
+        '',
+        ''
```

Comment:
this is a duckdb/athena difference in parsing that hadn't come up since we hadn't been running this study locally.